.github/workflows/rector.yaml

```diff
@@ -19,20 +16,15 @@ jobs:
         runs-on: ubuntu-latest
         timeout-minutes: 8
 
+        if: github.event.pull_request.head.repo.full_name == 'rectorphp/rector-src'
         steps:
-            # workaround for missing secret in fork PRs - see https://github.com/actions/checkout/issues/298
-            # see https://github.com/rectorphp/rector/commit/d395e1c28b8e6a56711dcc2e10490a82965850e4
             -
-                if: github.event.pull_request.head.repo.full_name == github.repository
                 uses: actions/checkout@v3
                 with:
                     # Must be used to trigger workflow after push
                     token: ${{ secrets.ACCESS_TOKEN }}
 
-            # in forks, the token is not available - so we cannot use it
-            -
-                if: github.event.pull_request.head.repo.full_name != github.repository
-                uses: actions/checkout@v3
+            - run: echo "run on ${{ github.event.pull_request.head.repo.full_name }}"
```

Comment:
@TomasVotruba this is to verify the repo name to verify and it show:

```
Run echo "run on rectorphp/rector-src"
run on rectorphp/rector-src
```

see https://github.com/rectorphp/rector-src/actions/runs/4316268191/jobs/7531815857#step:3:1

Reply:
The `pull_request.head` seems goes to forks repo name
while `github.repository` goes to target repo name, so I changed to `github.event.pull_request.head.repo.full_name`